docs/codeql/codeql-cli/creating-codeql-databases.rst

```diff
@@ -63,7 +63,9 @@ more than one language:
 - ``--no-run-unnecessary-builds``: used with ``--db-cluster`` to suppress the build 
   command for languages where the CodeQL CLI does not need to monitor the build 
   (for example, Python and JavaScript/TypeScript).
-   
+- extractor options: used to customize the behavior of extractors that create CodeQL databases. For more information,
```

Comment:
I went back and forth on this. Since the extractor options are not an option themselves, I don't think `extractor-options` would be accurate. But it's different enough that I might make extractor options its own small paragraph.